.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 .cache
+.idea
```

Comment:
is this change needed

Reply:
this removes project files

Reply:
I would run this one by the ADS team, as this repo is shared by the entire product so this may have unintended consequences.

+ @Charles-Gagnon any thoughts?

Reply:
I don't think anyone uses IntelliJ and not sure why anyone would for ADS. That being said I don't really have a problem with adding it if someone is actually using it.

@kisantia @kburtram FYI

Reply:
is this change actually necessary for this PR? If it isn't related to the other changes in this PR, please send a separate PR with an explanation for changing something like this.

Reply:
These unrelated changes rolled back